wolfcrypt/src/pkcs12.c

```diff
@@ -670,6 +670,33 @@ int wc_d2i_PKCS12(const byte* der, word32 derSz, WC_PKCS12* pkcs12)
     return ret;
 }
 
+/* Convert WC_PKCS12 struct to allocated DER buffer.
+ * pkcs12 : non-null pkcs12 pointer
+ * der    : pointer-pointer to der buffer holding PKCS12
```

Comment:
Where this is slightly different then most wolfCrypt API in that it mallocs a DER buffer and returns the pointer the free'ing responsibility should be noted in the comment here. I could see potential cases where engineers later using this function would unintentionally cause memory leaks.

Reply:
If we can avoid mallocing the DER buffer I think it would be preferred

Reply:
I added a check for a der pointer being passed in. It now operates as the OpenSSL man describes: Use passed in der buffer, otherwise allocate.